src/cheri_insts.sail

```diff
@@ -431,6 +474,21 @@ function clause execute (CSetAddr(cd, cs1, rs2)) = {
 
 
 union clause ast = CSetBounds : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is set to capability register *cs1* with its
+ * **base** field replaced with *cs1*.**address** and its **length** field
+ * replaced with integer register *rs2*. If the resulting capability cannot be
+ * represented exactly the bounds are padded to be representable but will never
+ * exceed the bounds of *cs1*.
```

Comment:
The MIPS description has a bit more detail here. Should we add more on the rounding behaviour?
I think the current description is fine for now, but we should included more information when we update the spec to have MIPS as the appendix.

Reply:
It's very difficult to describe precisely and accurately without just regurgitating all of CHERI Concentrate but I will add a bit more like CRRL and CRAM have.